src/pages/new/volunteer/index.js

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 
+import ArticleSpotlightCard from "../../../components/grid-aware/ArticleSpotlightCard";
+import articleSpotlightImage from "../../../components/grid-aware/ArticleSpotlightCard/laura-barerra-vera.png";
```

Comment:
For similar reasons to my last comment, could you also move this image into the actual page directory?